tools/scoring/score/parse_config.py

```diff
@@ -125,3 +125,37 @@ def match_reporting_entities(
           matches.append(proposed_entity.cloud_device_id)
 
     return matches
+
+  @staticmethod
+  def retrieve_reporting_translations(
+      *, matches: List[str], proposed: Dict[str, EntityInstance],
```

Comment:
can you please detail what is the input in the docstring.
matches is a list of cloud device id for matched entities.
proposed ...